src/partisan_hiscamp_membership_strategy.erl

```diff
@@ -97,7 +95,7 @@ distance(#hiScamp{membership=Membership0, heap=Heap, level1=L1, level2=L2, dista
 
 get_centroid_two_clusters(#hiScamp{membership=Membership0, heap=Heap, level1=L1, level2=L2} =State0) ->
     Min1 = heaps:min(Heap),
-    Heap = heaps:delete_min(Heap),
+    heaps:delete_min(Heap),
```

Comment:
This won't work either: you're returning the heap with an element deleted out of it and throwing the return value away.

Reply:
I'm a bit confused as to how the returned value is being thrown away? I'm assigned the first minimum to Min1 and the second to Min2 and then comparing them. Or are you saying I'm throwing away the return value of delete_min(Heap)?

Reply:
Yes, you are throwing away the return value of ```heaps:delete_min.```